src/outputtype.cpp

```diff
@@ -65,12 +66,13 @@ CTxDestination GetDestinationForKey(const CPubKey& key, OutputType type)
 std::vector<CTxDestination> GetAllDestinationsForKey(const CPubKey& key)
 {
     PKHash keyid(key);
+    CTxDestination p2pkh = PKHash(keyid);
```

Comment:
`keyid` is already of type `PKHash`. I guess you wanted to write

```suggestion
    CTxDestination p2pkh{keyid};
```

Reply:
Of course, that's much more idiomatic. Done.